.dockerignore

```diff
@@ -1,4 +1,3 @@
-target
```

Comment:
I'm not sure if we should remove this though, otherwise, we'd have to copy the target folder into the image. This slows down the build process because this folder would be 10s of GBs

Reply:
Gotcha, didn't know that. Patched it in order to exclude the target folder excluding the executable files for tesseract and telemetry if that helps

Reply:
Nice, also add one for the hyperbridge binary and should be good to merge